src/errors/test-run/index.js

```diff
@@ -43,3 +48,43 @@ export class ActionBooleanOptionError extends ActionOptionErrorBase {
     }
 }
 
+
+// Test execution client errors
+export class UncaughtJSError extends TestRunErrorBase {
```

Comment:
Let's name it `UncaughtErrorOnPage` since we will have `UncaughtErrorInTestCode`